src/api/wix/WixToolset.Data/Symbols/WixBundleSymbol.cs

```diff
@@ -243,14 +250,52 @@ public WixBundleCommandLineVariables CommandLineVariables
             set => this.Set((int)WixBundleSymbolFields.CommandLineVariables, (int)value);
         }
 
-        public PackagingType DefaultPackagingType => (this.Compressed.HasValue && !this.Compressed.Value) ? PackagingType.External : PackagingType.Embedded;
-
-        public bool DisableModify => (this.Attributes & WixBundleAttributes.DisableModify) == WixBundleAttributes.DisableModify;
+        public WixBundleModifyType DisableModify
+        {
+            get
+            {
+                if (Enum.TryParse((string)this.Fields[(int)WixBundleSymbolFields.DisableModify], true, out WixBundleModifyType value))
```

Comment:
We've switched to using the numeric values for enums instead of the string values underlying the Data API. I believe there are a couple of other stragglers left in the Bundle symbols if you wanted to grab them as part of this change, that'd be awesome.